docs/guides/how-to-guides/build-your-first-console.md

```diff
@@ -0,0 +1,469 @@
+---
+title: Build your first console
+
+---
+
+# Build your first console
+
+This how-to-guide will take you through the steps of defining a resource service all the way through to consuming the service on a frontend application. 
+
+## Prerequisites
+To get started with this guide, there are a few things you need to have done as an Alis Builder:
+
+1. Ensure you have installed and set up the [Alis CLI](../getting-started/command-line-interface.md)
+2. Ensure you have [created a product on the Alis Build platform](../getting-started/conceptual-framework.md)
+3. Ensure you have been added to the relevant organisation's developers group. Request to be added by the owner of the
```

Comment:
Probably need to swop around 2 and 3? Need to be added as developer before you can create product?